cloud/aws/README.md

```diff
@@ -129,12 +136,22 @@ provider "aws" {
 - You need to be an IAM admin on AWS account
 - The apply will wait between the AWS policy attachment to role and the signalfx aws integration creation to prevent permission denied error
 - This module does not support `services` and `custom_cloudwatch_namespaces` because `namespace_sync_rule` and `custom_namespace_sync_rule` are respectively more powerful but in conflict
+
+### CloudWatch Metric Streams
+
 - When enabling AWS Cloudwatch Metric Streams with `use_metric_streams_sync = true`, you also need to set `create_metric_streams_iam` to `true`. Then you need to create the Kinesis Data Firehose resources by yourself in each of the regions where you want to collect metrics from. You can use the [`aws-cloudwatch-metric-stream`](https://github.com/claranet/terraform-signalfx-integrations/tree/master/cloud/aws-cloudwatch-metric-stream) module to do this.
 - When disabling AWS Cloudwatch Metric Streams, make sure to apply in two phases:
   - first change the `use_metric_streams_sync` parameter from `true` to `false` and run `terraform apply` to let it deprovision the AWS Cloudwatch Metric Streams resources it created,
   - then change `create_metric_streams_iam` from `true` to `false` and run `terraform apply` to destroy the IAM role.
     If you do not follow that process, the AWS integration will end up in `CANCELATION_FAILED` status.
 
+### CloudWatch Logs sync (BETA)
+
+- When enabling AWS Cloudwatch Logs sync with `enable_logs_sync = true`, you also need to set `create_logs_iam` to `true`. 
```

Comment:
Don't you need an additional module to deploy the Splunk AWS Log Collector Lambda function used by this solution in each region, like we did for AWS Cloudwatch Metric Streams with [`aws-cloudwatch-metric-stream`](https://github.com/claranet/terraform-signalfx-integrations/tree/master/cloud/aws-cloudwatch-metric-stream)?

https://docs.splunk.com/Observability/gdi/get-data-in/connect/aws/aws-wizardconfig.html#choose-a-cloudformation-template

 https://o11y-public.s3.amazonaws.com/aws-cloudformation-templates/release/template_logs_regional.yaml

Reply:
of course we will need a separate module like for metric streams. This PR focus on bringing all features of the integration itself to be future proof. I cannot implement full support for logs for now while we don't have access to log observer yet but this should at least prepare its integration when available.

Reply:
https://github.com/claranet/terraform-signalfx-integrations/issues/52 created